nevergrad/functions/multiobjective/core.py

```diff
@@ -89,4 +89,34 @@ def pareto_front(self) -> List[ArgsKwargs]:
             if should_be_added:
                 new_points.append((argskwargs, losses))
         self._points = new_points
-        return [p[0] for p in self._points]
+
+    def pareto_front(self, size: Optional[int] = None, method: Optional[str] = None) -> List[ArgsKwargs]:
+        """Pareto front, as a list of args and kwargs (tuple of a tuple and a dict)
```

Comment:
With the new documentation, we'll need to level up the docstrings :D :
```
"""Pareto front, as a list of args and kwargs (tuple of a tuple and a dict)

Parameters
------------
size:  int (optional)
    if provided, selects a subset of the full pareto front with the given maximum size
subset: str
    method for selecting the subset ("random, "loss-covering", "domain-covering")

Returns
--------
list
    the list of elements of the pareto front
"""
```

Reply:
Done.